docs/reference/settings.rst

```diff
@@ -498,6 +498,41 @@ PROMOTED_JOBS_AMOUNT
 
     2
 
+JOBS_FRONTPAGE_RANGE_DAYS
```

Comment:
I would make it a combined setting (like vocabulary or contacts)

JOBS_WIDGET = {
   'days': 14,
   'min': 3,
   'max': 10,
}